playbooks/common/openshift-cluster/upgrades/etcd/upgrade.yml

```diff
@@ -31,6 +31,15 @@
     # state, not manipulating anything
     changed_when: false
 
+  - name: Record containerized etcd version
+    command: runc exec etcd_container rpm -qa --qf '%{version}' etcd\*
```

Comment:
nit: This could cause a problem if other packages that start with ``etcd`` are added. Why not: ``rpm -qa --qf '%{version}' etcd``?